pkg/tagger/collectors/workloadmeta_main.go

```diff
@@ -152,12 +153,27 @@ func fargateStaticTags(ctx context.Context) map[string]string {
 
 	// EKS Fargate specific tags
 	if fargate.IsEKSFargateInstance() {
+		// eks_fargate_node
 		node, err := fargate.GetEKSFargateNodename()
 		if err != nil {
 			log.Infof("Couldn't build the 'eks_fargate_node' tag: %w", err)
 		} else {
 			tags["eks_fargate_node"] = node
 		}
+
+		// kube_cluster_name
+		clusterTagName := "kube_cluster_name"
+		tag, found := tags[clusterTagName]
+		if found {
+			log.Infof("'%s:%s' was set manually via DD_TAGS, not changing it", clusterTagName, tag)
+		} else {
+			cluster := clustername.GetClusterName(context.TODO(), "")
```

Comment:
It seems we have received a `context.Context` as parameter.

```suggestion
			cluster := clustername.GetClusterName(ctx, "")
```